public/Lab-Files/Lesson02/Challenges/images.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>images</title>
+</head>
+<body>
+    <h1 style="margin-left:190px">The perfect trip</h1>
+    <br>
+    <p style="margin-left:200px">I would go to... <strong>Japan!</strong></p> 
+
+    <img src="./images/japan.jpg" alt="A picture of Japan" title="Japan"
+    width="700" height="300" />
+    
+    <p style="margin-left:150px">I would see scenes like... <strong>Tokyo Night Life!</strong></p> 
+
+    <img src="./images/Tokyo.jpg" alt="A picture of a busy Tokyo street" title="Tokyo"
+   width="700" height="300" />
+   
+    <p style="margin-left:150px">And do things like...<strong>eat world class sushi!</strong></p> 
+
+    <img src="./images/sushi.jpg" alt="A picture of delicious sushi" title="A picture of delicious sushi"
+    width="700" height="300" />
+    
+    
+</body>
+</html>
```

Comment:
Awesome job. For the most part, your code is clean and readable. So far I can tell you have been validating your code too.